### ISSUE TITLE
Migrate From Backup fails with an SQL file (uncompressed) created by Geeklog DBBackUp feature

### ISSUE THREAD

**Issue by mystralkk** [Geeklog-Core/geeklog#987]
The Installer fails with the following message.
```
Processing file: C:/Users/mystr/Documents/Computer/http_docs/geeklog/backups/geeklog_db_backup_2019_11_09_10_05_13.sql

NOTICE Error at the line 3272: INSERT INTO `gl_stories` VALUES ('welcome', 2, 0, '2019-11-09 10:04:07', '', 'Welcome to Geeklog!', '', '
(snip)
', '', 1, 100, 1, 0, '', 0, '', 1, 1, 0, 0, 0, 0, '', 'html', 0, 1, '', '', 0, 2, 3, 3, 2, 2, 2);.
Query: INSERT INTO `gl_stories` VALUES ('welcome', 2, 0, '2019-11-09 10:04:07', '', 'Welcome to Geeklog!', '', '<p>Welcome and let me be the first to congratulate you on installing Geeklog. Please take the time to read everything in the <a href=\"docs/english/index.html\">docs directory</a>. Geeklog now has enhanced, user-based security. You should thoroughly understand how these work before you run a production Geeklog Site.</p>\r<p>To log into your new Geeklog site, please use this account:</p>\r<p>Username: <b>Admin</b><br />\rPassword: <b>password</b></p><p><b>And don\'t forget to <a href=\"usersettings.php\">change your password</a> after logging in!</b></p>', '', 1, 100, 1, 0, '', 0, '', 1, 1, 0, 0, 0, 0, '', 'html', 0, 1, '', '', 0, 2, 3, 3, 2, 2, 2);

MySQL: Incorrect datetime value: '' for column 'modified' at row 1
```
It seems that this is because DBBackUp outputs a NULL value as an empty string'' not NULL.

**Comment by mystralkk** [comment]
Fixed with change set 0588312.

**Comment by eSilverStrike** [comment]
Tried Database backup and I got an error inserting this record. The 2 time columns at the end do not have quotes around them.

`INSERT INTO `gl_events` VALUES ('20180213155639544', 'Test', 'TEst', 'html', '2018-02-13', '2018-02-13', '', 0, 2, 18, 3, 2, 2, 2, '', '', '', '', '', 0, '', 'TEst', 15:45:00, 15:45:00);`

The record before inserted correctly as the last 2 columns where Null.

**Comment by mystralkk** [comment]
Additional fix with change set 5ebf311.

**Comment by eSilverStrike** [comment]
Fix works. Was able to complete a migration process by using a Geeklog Database backup (the database did have a pile of test data including none core plugins).
